src/Annotation/Admin.php

```diff
@@ -23,6 +23,9 @@
  *
  * @Annotation
  * @Target("CLASS")
+ *
+ * @deprecated since 3.x, to be removed in 4.0.
+ * @see https://github.com/kunicmarko20/SonataAutoConfigureBundle
```

Comment:
AutoConfigure or SonataAnnotations?

Reply:
AutoConfigure. They are similar but different. SonataAnnotations allows you to not create admin classes at all and put annotations on your entity. AutoConfigure is similar to the thing we had here since we have an annotation for admin class, the difference is annotation is not required and AutoConfigure will try to guess everything.